app/controllers/idv/otp_verification_controller.rb

```diff
@@ -70,6 +71,26 @@ def handle_otp_confirmation_failure
       end
     end
 
+    def save_in_person_notification_phone
+      return unless IdentityConfig.store.in_person_send_proofing_notifications_enabled
+      return unless is_in_person_session?
+      # future tickets will support voice otp
+      return unless idv_session.user_phone_confirmation_session.delivery_method == :sms
+
+      current_enrollment.notification_phone_configuration =
+        NotificationPhoneConfiguration.new(
+          phone: idv_session.user_phone_confirmation_session.phone,
+        )
+    end
+
+    def current_enrollment
+      current_user.establishing_in_person_enrollment
+    end
+
+    def is_in_person_session?
```

Comment:
```suggestion
    def in_person_enrollment?
      return false unless IdentityConfig.store.in_person_proofing_enabled
```
Could follow [the pattern](https://github.com/18F/identity-idp/blob/main/app/controllers/idv/personal_key_controller.rb#L84-L87) in `Idv::PersonalKeyController` here.